documentation/modules/rn-2.6.adoc

```diff
@@ -2,6 +2,287 @@
+
+
+.The `must-gather` logs can now be loaded only by using the CLI
+
+The {product-short} web console can no longer download logs. With this update, you must download `must-gather` logs by using CLI commands. For more information, see link:https://github.com/openshift/must-gather-operator[Must Gather Operator].
+
+
+.Migrated VMs can now be reviewed and configured
+
+The {project-short} web console now displays the name and migration status of each virtual machine (VM). The migrated VMs have the `app: <vmname>` label to help further changes in {ocp-name} Virtualization.
+The console displayed those details before this change. The only difference is that the VM is set with the `app: <vmname>` labels, which makes it easier to make further configurations related to the VM on the target cluster
+link:https://issues.redhat.com/browse/MTV-843[(MTV-843)]
+
+
+.{project-short} no longer runs `pvc-init` pods when migrating from vSphere
+
+.{project-short} no longer runs `pvc-init` pods when migrating from vSphere. However, the CDI does runs `pvc-init` pods as they are included in the `cdi.kubevirt.io/storage.bind.immediate.requested` annotation on the data volume.
+
+
+[id="new-features-and-enhancements-26_{context}"]
+== New features and enhancements
+
+This part provides features and enhancements introduced in {project-full} 2.6:
+
+.Validation for RHEL 6 guest operating system is now available when migrating VMs with RHEL 6 guest operating system
+
+Red Hat Enterprise Linux (RHEL) 9 does not support RHEL 6 as a guest operating system. Therefore, RHEL 6 is not supported in {ocp-name} Virtualization. With this update, a validation of RHEL 6 guest operating system was added to {ocp-name} Virtualization. link:https://issues.redhat.com/browse/MTV-413[(MTV413)]
+
+.Migrations from {ocp-name} versions before 4.13 are now blocked
+
+The `Export API` function in {ocp-name} Virtualization now exposes VM manifests from {ocp-name} 4.13. Therefore, source {ocp} clusters must be version 4.13 or later. link:https://issues.redhat.com/browse/MTV-734[(MTV-734)]
+
+
+.Automatic retrieval of CA certificates for the provider's URL in the console
+
+The ability to retrieve CA certificates, which was available in previous versions, has been restored. The `vSphere Verify certificate` option is in the `add-provider` dialog. This option was removed in the transition to the {ocp} console and has now been added to the console. This functionality is also available for RHV, OpenStack and OpenShift providers now
+link:https://issues.redhat.com/browse/MTV-737[(MTV-737)]
+
+
+.{project-short} UI plugin now uses only a dynamic plugin for the SDK REST API `fetch` methods
+
+The UI plugin now uses a dynamic plugin for the SDK REST API when using the `fetch` methods. link:https://issues.redhat.com/browse/MTV-616[(MTV-616)]
+
+
+
```

Comment:
@ahadas - please can you give us a bit more information about this ticket?
Thanks